catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -303,14 +297,16 @@
                             ddf.catalog.resourceretriever,
                             ddf.catalog.source.impl,
                             ddf.catalog.util.impl,
-                            org.codice.ddf.catalog.resource.cache,
                             org.codice.ddf.catalog.resource.cache.impl,
+                            org.codice.ddf.catalog.resource.download.impl,
```

Comment:
I agree with you.  Unfortunately the classes don't get included in the bundle if they are not private packaged.  It looks like others have experienced this problem also since there are packages in this module that are also private packaged.  I'm not sure if there is some issue with the bnd tool and our naming.